bftengine/src/bftengine/DebugStatistics.cpp

```diff
@@ -141,28 +141,11 @@ void DebugStatistics::clearCounters(DebugStatDesc& d) {
   d.pendingRequests = 0;
 }
 
-#ifdef USE_TLS
```

Comment:
Yes, there is no ThreadLocalData class in the repo, so my guess it that someone removed it and because the flag was always set to false in that file the programmer didn't notice.